frontend/src/App.js

```diff
@@ -1,22 +1,82 @@
 import "./App.css";
 import Kitchen from "./pages/Kitchen.js";
 import logo from "./logo.svg";
+import { useState } from "react";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import Navbar from "./components/Navbar";
-import NutritionFacts from './components/NutritionFacts';
+import Recipes from "./pages/Recipes.jsx";
+import NutritionFacts from "./components/NutritionFacts";
+import ingredients from "./data/ingredients.json";
+
+const recipes = ingredients["ingredients by recipe"];
 
 function App() {
+  const [days, setDays] = useState({
+    Monday: [],
+    Tuesday: [],
+    Wednesday: [],
+    Thursday: [],
+    Friday: [],
+    Saturday: [],
+    Sunday: [],
+  });
+  const [isDragging, setIsDragging] = useState(false);
+  const smoothies = Object.keys(recipes).map((recipe, index) => {
+    return {
+      id: index,
+      text: recipe,
+      img: recipes[recipe]["imageURL"],
+    };
+  });
+  const recipeProps = {
+    days,
+    isDragging,
+    handleRemove,
+    handleDragStart,
+    handleDragEnd,
+    smoothies,
+  };
+
   return (
     <div className="App">
       <BrowserRouter>
         <Navbar />
         <Routes>
-          <Route index element={<div>recipe screen</div>}/>
-          <Route path="/kitchen" element={<Kitchen />}/>
+          <Route index element={<Recipes {...recipeProps} />} />
```

Comment:
ah good i was wondering how arguments could be passed through components like this with props